scripts/db/schema-5.down.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
+
+ALTER TABLE "study_requests" DROP COLUMN "assignedTo";
+
+DROP TABLE "study_request_assignees";
+
+DELETE FROM "study_request_reasons" WHERE "value" = 'AWS';
```

Comment:
can we get rid of this `AWS` reason?  these should probably be rethought, and randomly adding individual reasons isn't a good way to do that.  also, this doesn't relate to the larger change here in any meaningful way.